doc: add meeting minutes Aug 22 (#1153)
openjs-foundation/cross-project-council: meetings/2023/2023-08-22.md

# OpenJS Foundation Cross Project Council Meeting 2023-08-22

## Links

* **Recording**: https://www.youtube.com/watch?v=u9Zu_f0ucq0
* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/1151

## Present

* Michael Dawson (@mhdawson)
* Joe Sepi (@joesepi)
* Ethan Arrowood (@ethan-arrowood)
* Robin Ginn (@rginn)
* Tobie Langel (@tobie)
* Ben Sternthal (@bensternthal)
* Adrian Estrada(@edsadr)
* Rick Markins (@rxmarbles)
* Claudio Wunder (@ovflowd)

> * Sara Chipps (@SaraJo)

Reviewer: ```suggestion
* Sara Chipps (@SaraJo)
* Jordan Harband (@ljharb)
```

# OpenJS Foundation Cross Project Council Meeting 2023-08-22

## Links

* **Recording**: https://www.youtube.com/watch?v=u9Zu_f0ucq0
* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/1151

## Present

* Michael Dawson (@mhdawson)
* Joe Sepi (@joesepi)
* Ethan Arrowood (@ethan-arrowood)
* Robin Ginn (@rginn)
* Tobie Langel (@tobie)
* Ben Sternthal (@bensternthal)
* Adrian Estrada(@edsadr)
* Rick Markins (@rxmarbles)
* Claudio Wunder (@ovflowd)
* Sara Chipps (@SaraJo)
* Jordan Harband (@ljharb)

## Agenda

### Announcements

* Standards meeting happening after this meeting
* Collab Summit coming up!

### Board Meeting Updates

> *Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.

Reviewer 2: ```suggestion
* Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
```

### openjs-foundation/cross-project-council

* Revamping the travel fund [#1147](https://github.com/openjs-foundation/cross-project-council/issues/1147)

>   * speakers have most priority, outside of that we dont have criteria for when to approve or not
>   * board is aware, budget extension would be tough given broader context
>   * CPC should consider as part of this discussion if more funds should be requested in 2023
>   * will do a working session next week, ben will prep materials

Reviewer 2: ```suggestion
  * speakers have the most priority, outside of that we don't have criteria for when to approve or not
  * Board is aware, the budget extension would be tough given the broader context
  * CPC should consider as part of this discussion if more funds should be requested in 2023
  * will do a working session next week, Ben will prep materials
```

* Pin project-status repo to landing page [#1140](https://github.com/openjs-foundation/cross-project-council/issues/1140)
 * Done

* Requirements for Projects releasing internal projects to 3rd party contributor [#1133](https://github.com/openjs-foundation/cross-project-council/issues/1133)
  * Waiting on legal

* Requirements for Projects ingesting other existing external projects [#1059](https://github.com/openjs-foundation/cross-project-council/issues/1059)
  * Waiting on legal

* Enable ability to have pseudonimity for EasyCLA. [#1088](https://github.com/openjs-foundation/cross-project-council/issues/1088)
  * Waiting on legal

* 2023 CPC Summer Elections [#1126](https://github.com/openjs-foundation/cross-project-council/issues/1126)
  * Discussed and raised awareness
  * nominations are happening

* Code of Conduct Plan of Action [#1122](https://github.com/openjs-foundation/cross-project-council/issues/1122)

>   * not much in last couple weeks but more soon

Reviewer 2: ```suggestion
  * not much in the last couple of weeks but more soon
```

* Standardize on Collab Spaces over Working Groups [#1110](https://github.com/openjs-foundation/cross-project-council/issues/1110)
  * CPC Director to decide when to give a courtesy update to the board about the expected non-contentious charter changes

* Non-violent communication / strategic workplace conflict resolution training? [#1106](https://github.com/openjs-foundation/cross-project-council/issues/1106)
  * Robin still working on this

* JSON Schema charter request to review [#1056](https://github.com/openjs-foundation/cross-project-council/issues/1056)
  * Will discuss more

* Simplify CPC Board representation [#1055](https://github.com/openjs-foundation/cross-project-council/pull/1055)

> * working with legal council to get final by-laws changes, following that CPC changes will be made and proposed to board

Reviewer 2: ```suggestion
* working with legal council to get final by-laws changes, following that CPC changes will be made and proposed to the board
```

* Open Up Community Board Seat [#1054](https://github.com/openjs-foundation/cross-project-council/issues/1054)
  * bundled with #1055

* Should we have a co-chair? [#1043](https://github.com/openjs-foundation/cross-project-council/issues/1043)
  * bundled with #1055

* Balancing privacy vs. transparency for the travel fund [#397](https://github.com/openjs-foundation/cross-project-council/issues/397)
  * Will work on this as a part of the working session next week on #1147

## Next week's working session

> * Figure out Travel Fund funding, process, 2024 plan and such #1147

Reviewer 2: ```suggestion
* Figure out Travel Fund funding, process, 2024 plan, and such #1147
```
